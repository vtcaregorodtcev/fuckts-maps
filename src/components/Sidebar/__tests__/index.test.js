import React from 'react';
import renderer from 'react-test-renderer';
import { Sidebar } from '../index';

it('renders correctly', () => {
  const tree = renderer.create(<Sidebar maps={[]} />).toJSON();
  expect(tree).toMatchSnapshot();
});
