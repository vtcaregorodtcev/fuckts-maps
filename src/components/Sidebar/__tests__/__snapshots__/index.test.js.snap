// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<aside
  className="ant-layout-sider ant-layout-sider-dark ant-layout-sider-collapsed ant-layout-sider-has-trigger"
  style={
    Object {
      "flex": "0 0 80px",
      "maxWidth": "80px",
      "minWidth": "80px",
      "width": "80px",
    }
  }
>
  <div
    className="ant-layout-sider-children"
  >
    <div
      className="logo"
    />
    <ul
      className="ant-menu ant-menu-dark ant-menu-inline-collapsed ant-menu-root ant-menu-vertical"
      onMouseEnter={[Function]}
      onTransitionEnd={[Function]}
      role="menu"
      style={Object {}}
    >
      <li
        className="ant-menu-item"
        data-cy-sidebar-menu-item-new-map={true}
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={Object {}}
      >
        <i
          aria-label="icon: file"
          className="anticon anticon-file"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="file"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M854.6 288.6L639.4 73.4c-6-6-14.1-9.4-22.6-9.4H192c-17.7 0-32 14.3-32 32v832c0 17.7 14.3 32 32 32h640c17.7 0 32-14.3 32-32V311.3c0-8.5-3.4-16.7-9.4-22.7zM790.2 326H602V137.8L790.2 326zm1.8 562H232V136h302v216a42 42 0 0 0 42 42h216v494z"
            />
          </svg>
        </i>
        <span>
          Новая карта
        </span>
      </li>
      <li
        className="ant-menu-item"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={Object {}}
      >
        <i
          aria-label="icon: save"
          className="anticon anticon-save"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="save"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M893.3 293.3L730.7 130.7c-7.5-7.5-16.7-13-26.7-16V112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V338.5c0-17-6.7-33.2-18.7-45.2zM384 184h256v104H384V184zm456 656H184V184h136v136c0 17.7 14.3 32 32 32h320c17.7 0 32-14.3 32-32V205.8l136 136V840zM512 442c-79.5 0-144 64.5-144 144s64.5 144 144 144 144-64.5 144-144-64.5-144-144-144zm0 224c-44.2 0-80-35.8-80-80s35.8-80 80-80 80 35.8 80 80-35.8 80-80 80z"
            />
          </svg>
        </i>
        <span
          data-cy-save-map={true}
        >
          Сохранить карту
        </span>
      </li>
      <li
        className="ant-menu-item"
        data-cy-sidebar-menu-item-remove-map={true}
        onClick={[Function]}
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        role="menuitem"
        style={Object {}}
      >
        <i
          aria-label="icon: delete"
          className="anticon anticon-delete"
        >
          <svg
            aria-hidden="true"
            className=""
            data-icon="delete"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M360 184h-8c4.4 0 8-3.6 8-8v8h304v-8c0 4.4 3.6 8 8 8h-8v72h72v-80c0-35.3-28.7-64-64-64H352c-35.3 0-64 28.7-64 64v80h72v-72zm504 72H160c-17.7 0-32 14.3-32 32v32c0 4.4 3.6 8 8 8h60.4l24.7 523c1.6 34.1 29.8 61 63.9 61h454c34.2 0 62.3-26.8 63.9-61l24.7-523H888c4.4 0 8-3.6 8-8v-32c0-17.7-14.3-32-32-32zM731.3 840H292.7l-24.2-512h487l-24.2 512z"
            />
          </svg>
        </i>
        <span>
          Удалить карту
        </span>
      </li>
    </ul>
  </div>
  <div
    className="ant-layout-sider-trigger"
    onClick={[Function]}
    style={
      Object {
        "width": "80px",
      }
    }
  >
    <i
      aria-label="icon: right"
      className="anticon anticon-right"
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="right"
        fill="currentColor"
        focusable="false"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M765.7 486.8L314.9 134.7A7.97 7.97 0 0 0 302 141v77.3c0 4.9 2.3 9.6 6.1 12.6l360 281.1-360 281.1c-3.9 3-6.1 7.7-6.1 12.6V883c0 6.7 7.7 10.4 12.9 6.3l450.8-352.1a31.96 31.96 0 0 0 0-50.4z"
        />
      </svg>
    </i>
  </div>
</aside>
`;
